Calling/ClientApp/src/components/Configuration.tsx

```diff
@@ -60,7 +60,7 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
     setUserId(userId);
     initCallClient(userId, unsupportedStateHandler, endCallHandler);
     setGroup(groupId);
-  }, [userId, groupId, setUserId, initCallClient, setGroup, unsupportedStateHandler, endCallHandler]);
+  }, []);
```

Comment:
UserId looks like is changed by the HTML code below (line 99-106). If it changes do we need to re-run this useEffect with the new UserId? I'm not familiar with this code but maybe another option is to remove this useEffect completely and call the setUserId, initCallClient, and setGroup as part of the button onClick.

Reply:
None of the dependencies are changing. I recall we had this warning pop up if you were using variables in your useEffect block that were defined outside however it is quite annoying for the case of you just want useEffect to get kicked off on mount/unmount. 

We need the useEffect block since we initialize the deviceManager through the sdk with it. We use the existence of the deviceManager to determine if we could set up the SDK or not. So we need this useEffect block today.

I think we just didn't want this block of code to get fired in other scenarios and it was. So we are fixing this by ensuring this useEffect block is ONLY fired on mount/unmount scenarios now.